doc/radosgw/multisite.rst

```diff
@@ -921,6 +921,10 @@ because not all of the settings end up in a Ceph configuration file. You
 can list zone groups, get a zone group configuration, and set a zone
 group configuration.
 
+.. note:: For S3 clients, zone group maps to AWS regions, the value ``us-east`` works
+          as default value. If any other value need to used as AWS region then user need
+          to create equivalent zone group.
+
```

Comment:
rgw doesn't do anything special with 'us-east' or 'us-east-1', this default is probably specific to some s3 clients

where exactly does this region issue come up? afaik, rgw doesn't care what the region name says in v2/v4 auth, as long as the signature matches

rgw *does* use zonegroup api_name to match the [LocationConstraint](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateBucket.html#API_CreateBucket_RequestSyntax) field of CreateBucket requests, and that part is documented in https://docs.ceph.com/en/latest/radosgw/placement/#s3-bucket-placement

we really shouldn't recommend that users create zonegroups to match their clients

Reply:
@cbodley : Thanks for the expalination. Also can you please explain a bit about  `this default is probably specific to some s3 clients`, is it meant only for clients who access rgw via rest API like adminsOps?
I like to resolve this https://github.com/rook/rook/issues/9486 in clean manner, do you think it is better to fix it only in Rook docs than here?

Reply:
> Also can you please explain a bit about this default is probably specific to some s3 clients, is it meant only for clients who access rgw via rest API like adminsOps?

it's not specific to /admin APIs, no. those APIs do authenticate with aws signatures, but so does every other s3 request

i followed up with some details on the rook issue here: https://github.com/rook/rook/discussions/9142#discussioncomment-2083019